insights/specs/default.py

```diff
@@ -356,7 +356,6 @@ class DefaultSpecs(Specs):
     firewalld_conf = simple_file("/etc/firewalld/firewalld.conf")
     foreman_production_log = simple_file("/var/log/foreman/production.log")
     fstab = simple_file("/etc/fstab")
-    fw_devices = simple_command("/bin/fwupdagent get-devices", deps=[IsBareMetal])
```

Comment:
I note that the FwupdagentDevices depends on this spec, if it's Okay to stop collecting it?

Reply:
Not a problem yet. got it in the Jira issue